src/utils/common.js

```diff
@@ -91,70 +98,70 @@ export function formatMoney(
   }
 
   return (c ? num.replace('.', c) : num).replace(new RegExp(re, 'g'), `$&${s || ','}`);
-}
+};
 
-export function parseNumber(amount: string = '0') {
-  let strg = amount.toString() || '';
+export const parseNumber = (amount: string) => {
+  let strg = amount;
   let decimal = '.';
   strg = strg.replace(/[^0-9$.,]/g, '');
   if (strg.indexOf(',') > strg.indexOf('.')) decimal = ',';
   if ((strg.match(new RegExp(`\\${decimal}`, 'g')) || []).length > 1) decimal = '';
   strg = strg.replace(new RegExp(`[^0-9$${decimal}]`, 'g'), '');
   strg = strg.replace(',', '.');
+
   return parseFloat(strg);
-}
+};
 
-export function isValidNumber(amount: string = '0') {
-  const strg = amount.toString() || '';
+export const isValidNumber = (amount: string) => {
```

Comment:
Same as above, I would ensure first that we're passing the right param here

Reply:
Updated and added a couple of tests just in case.